### PR TITLE
feat: upgrade project to NGO v1.6.0 [MTT-6875]

### PR DESCRIPTION
**Pull request by fernando-cortez**
### Description
PR to upgrade the project to NGO v1.6.0
In the process an Version Define has been created for NGO versions 1.5.2 & 1.6.0 and a singular test has been blocked out for these versions.
An upcoming NGO upgrade will address failing Shutdown-related tests.

### Issue Number(s)
[MTT-6875](https://jira.unity3d.com/browse/MTT-6875)

### Contribution checklist
 - [ N/A ] Tests have been added for boss room and/or utilities pack
 - [x] Release notes have been added to the [project changelog](../CHANGELOG.md) file and/or [package changelog](../Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md) file
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] JIRA ticket ID is in the PR title or at least one commit message
 - [x] Include the ticket ID number within the body message of the PR to create a hyperlink
 - [ N/A ] An Index entry has been added in readme.md if applicable

### CODE REVIEW

**Review by LPLafontaineB — COMMENTED** [review]
Should also include toggling off the `Generate Default Network Prefabs List` parameter in the project settings now that it is available

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ Additional documentation and release notes are available at [Multiplayer Documen
 * Upgraded editor version to 2022.3.7f1 (#855)
   * Upgraded Authentication Service package to v2.7.1
   * Upgraded Wire Service package to v1.2.0
+* Upgraded Boss Room to Netcode for GameObjects v1.6.0 (#865)
+  * A package Version Define has been created for Netcode for GameObjects v.1.5.2 - v1.6.0. A subsequent NGO version upgrade will address the runtime test that has been ignored with this version bump
```
Comment (LPLafontaineB) [inline CHANGELOG.md:16]: Should include a brief explanation of the reason why this define is added

**Comment by fernando-cortez** [comment]
> Should also include toggling off the `Generate Default Network Prefabs List` parameter in the project settings now that it is available

It's been mentioned as a part of [this ](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/pull/828) release. Or is there something else to mention?

**Comment by LPLafontaineB** [comment]
> > Should also include toggling off the `Generate Default Network Prefabs List` parameter in the project settings now that it is available
> 
> It's been mentioned as a part of [this ](https://github.com/Unity-Technologies/com.unity.multiplayer.samples.coop/pull/828) release. Or is there something else to mention?

What I meant is the new setting in the Project Settings that makes it so it doesn't automatically generate a new DefaultPrefabsList. Now that it is available and that the bug related to it is fixed, we should update our project settings to use it.

File: CHANGELOG.md (hunk 1 above)
Comment (LPLafontaineB) [reply CHANGELOG.md]: Nitpick, but I think we could specify that this is specifically for a failing test, that the regression doesn't affect gameplay in the usual flow

**Review by LPLafontaineB — APPROVED** [review]

**Pull request merged**